src/OpenTK.Windowing.Desktop/GameWindow.cs

```diff
@@ -419,5 +432,35 @@ public void ResetTimeSinceLastUpdate()
         {
             _watchUpdate.Restart();
         }
+
+        private void Win32_OnModalSizeMoveBegin()
+        {
+            if (_win32SuspendTimerOnDrag)
+            {
+                _watchUpdate.Stop();
+            }
+        }
+
+        private void Win32_OnModalSizeMoveEnd()
+        {
+            if (_win32SuspendTimerOnDrag)
+            {
+                _watchUpdate.Restart();
+            }
+        }
+
+        /// <inheritdoc />
+        public override void Dispose()
+        {
+            base.Dispose();
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && !(_win32WndProc is null))
```

Comment:
Agreed, I prefer `is not null` too, but that requires C# version 9 (the projects are currently 7.3).

I'd gotten into the habit of using `is` even with `!` because where I work, people do stupid things like override operators in unexpected places, but obviously that won't apply here! I'll go with `!=` unless you're open to bumping the language version?